desc/integrals/_bounce_utils.py

```diff
@@ -126,7 +126,7 @@ def _check_spline_shape(knots, g, dg_dz, pitch_inv=None):
 def bounce_points(
     pitch_inv, knots, B, dB_dz, num_well=None, check=False, plot=True, **kwargs
 ):
-    """Compute the bounce points given spline of |B| and pitch λ.
+    """Compute the bounce points given spline of B and pitch λ.
```

Comment:
I am assuming the bounce points are calculated per field line so it's a 1D calculation, right?
Please add that to the description because I might play around with a 2D intersection idea later.

Reply:
done in new PR